scripts/upgrade

```diff
@@ -180,13 +180,23 @@ fi
 # SECURE FILES AND DIRECTORIES
 #=================================================
 
-# Restrict rights to Wallabag user only
-chmod 600 $wb_conf
+if [ "$upgrade_type" == "UPGRADE_APP" ]
+then
+	# Restrict rights to Wallabag user only
+	chmod 600 $wb_conf
+fi
```

Comment:
Why only in that situation ?

Reply:
I have no idea. What does that condition checks ?
@kay0u ?

Reply:
UUh that's a good question!

Reply:
Ok.
But then, why **only** if the script recreate the file ?
We usually set the permission back on all files, just in case an user have played with the permission on the files.
We know that chmod -R 777 are common...

And here, you especially change the behavior to add a condition.

Reply:
It's true, to be honest, I don't remember if I had a very real reason for adding this condition.

Reply:
let's wait the ci [![Build Status](https://ci-apps-dev.yunohost.org/jenkins/job/wallabag2_ynh%20PR76%20(Kay0u)/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wallabag2_ynh%20PR76%20(Kay0u)/)

Reply:
Anyway, I was about to say, do as you prefer.
But you already revert.

Both situation would be ok, but if we change, we should know why I think.